src/foremast/templates/pipeline/trigger-jenkins.json.j2

```diff
@@ -4,5 +4,6 @@
       "type": "jenkins",
       "master": "JenkinsCI",
       "job": "{{ data.app.triggerjob }}",
+      "runAsUser": "{{ data.app.run_as_user }}",
```

Comment:
For users that are not using fiat, the current default of `''` will likely cause issues. I see @e4r7hbug thought on making this a `null`/`None` default.
Alternatively, its creating a jinja conditional if `data.app.run_as_user` is populated or not.